examples/connect.rs

```diff
@@ -14,7 +14,13 @@ use tokio_stomp_2::*;
 
 #[tokio::main]
 async fn main() -> Result<(), anyhow::Error> {
-    let conn = client::connect("127.0.0.1:61613", None, None).await?;
+    let conn = client::connect(
+        "127.0.0.1:61613",
+        "/".to_string(),
+        "guest".to_string().into(),
```

Comment:
Not sure if it has any practical difference, but the change of None to "guest" moves any default behaviour to the client side instead of the server side.